core/amber/src/main/resources/python_udf/svm_classifier.py

```diff
@@ -11,27 +11,24 @@ class SVMClassifier(texera_udf_operator_base.TexeraMapOperator):
     def __init__(self):
         super(SVMClassifier, self).__init__(self.predict)
         self._model_file_path = None
-        self._model = None
         self._vc = None
+        self._clf = None
 
     def open(self, *args):
         super(SVMClassifier, self).open(*args)
         self._model_file_path = args[-1]
+        with open(self._model_file_path, 'rb') as file:
+            self._vc, self._clf = pickle.load(file)
```

Comment:
can you add some comments to say how these changes make the reading parallel?

Reply:
we used different serialize libraries. previously `shelve` does not support concurrent read, changed to `pickle` to allow concurrent read. I think this is level of python libraries, does not need to be added to source code?